frontend/views/components/Modal/ModalMixins.js

```diff
@@ -0,0 +1,51 @@
+import sbp from '~/shared/sbp.js'
+import { CLOSE_MODAL, UNLOAD_MODAL } from '~/frontend/utils/events.js'
+import ModalClose from './ModalClose.vue'
+
+const modaMixins = {
+  data () {
+    return {
+      isActive: true
+    }
+  },
+  components: {
+    ModalClose
+  },
+  mounted () {
+    sbp('okTurtles.events/on', CLOSE_MODAL, this.hide)
+    window.addEventListener('keyup', (e) => {
+      if (e.key === 'Escape') {
+        this.close()
+      }
+    })
+  },
+  beforeDestroy () {
+    sbp('okTurtles.events/emit', UNLOAD_MODAL)
+    window.removeEventListener('keyup', null)
```

Comment:
Hmm, am I right in understanding that passing `null` will remove *all* `keyup` listeners currently registered on window?

That's not what we want to do here though, right? Instead it should only remove the specific listener you registered. I think you can do that by moving the definition of the listener on line 16 up above line 5, and assigning it to a variable, then referencing that variable on lines 16 and here (24).